service/history/handler.go

```diff
@@ -742,6 +742,9 @@ func (h *Handler) ImportWorkflowExecution(ctx context.Context, request *historys
 		return nil, h.convertError(err)
 	}
 
+	if !shardContext.GetClusterMetadata().IsGlobalNamespaceEnabled() {
```

Comment:
Anything will break if the global namespace is disabled?

Reply:
nil pointer exception, plus ndc related logic may not be initialized